### ISSUE TITLE
Installation Error due to "sklearn" package name

### ISSUE THREAD

**Issue by ulf1** [fzi-forschungszentrum-informatik/TSInterpret#13]
**Describe the bug**

```
Collecting sklearn
  Downloading sklearn-0.0.post1.tar.gz (3.6 kB)
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  Preparing metadata (setup.py) ... error
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**To Reproduce**
Steps to reproduce the behavior. Also State the OS and the versions you are using.

run in a Colab notebook
```
!pip install TSInterpret
```

**Additional context**

**Comment by JHoelli** [comment]
Hi, 
thanks for the feedback. For me it was not possible to replicate this issue in google colab. 
!pip install TSInterpret works fine with python 3.9 for me. Can you maybe give a few more details (e.g., Python vesion, GPU/ CPU ) ? 

Thank a lot in advance. 
Best 
Jacqueline

**Comment by ulf1** [comment]
> !pip install TSInterpret

https://colab.research.google.com/drive/1cq-Ku9Owbkf76RExVEuZS0gIedSKGhgu?usp=sharing

**Comment by ulf1** [comment]
it works with 
```
!git clone https://github.com/fzi-forschungszentrum-informatik/TSInterpret.git
!cd TSInterpret && pip install -e .
```
but this isn't great as many people consider tools like Colab to tinker with new tools.

**Comment by JHoelli** [comment]
For Completeness: 

Issue also occurs when installing TSInterpret in python 3.7. due to deprecation of sklearn. 
Can currently not be solved for python 3.7. due to dependencies of used packages (e.g., mkdocs relies on sklearn).

Preferably use python >= 3.9. .
